x/commitment/types/keys.go

```diff
@@ -14,6 +14,15 @@ const (
 	MemStoreKey = "mem_commitment"
 )
 
+const (
+	BaseDenom       = "base-denom"
+	VestingDenom    = "vesting-denom"
+	EpochIdentifier = "epoch-identifier"
+	NumEpochs       = "num-epochs"
+	VestNowFactor   = "vest-now-factor"
+	NumMaxVestings  = "num-max-vestings"
```

Comment:
The CLI flags will need to be defined on cli package, not on types/keys.

Reply:
Done